packages/application-shell/src/test-utils/test-utils.tsx

```diff
@@ -416,7 +426,16 @@ function renderApp<AdditionalEnvironmentProperties = {}>(
           >
             <Router history={history}>
               <React.Suspense fallback={<LoadingFallback />}>
-                {props.children}
+                <ApplicationEntryPoint
+                  environment={mergedEnvironment}
+                  render={
+                    renderEntryPointRoutes
+                      ? undefined
+                      : () => <>{props.children}</>
```

Comment:
Should we exact this? It unfortunate to wrap `props.children` in a render-prop. Maybe `const renderOrChildren = { !renderEntryPointRoutes ? children : render: undefined } to then `...renderOrChildren`? Not entirely sure myself.

Reply:
I think I'll keep it like this, it's a bit more explicit and things stay simple.